internal/dump/common.go

```diff
@@ -74,13 +75,21 @@ func dumpTree(ctx context.Context, repo restic.Repository, rootNode *restic.Node
 // GetNodeData will write the contents of the node to the given output.
 func GetNodeData(ctx context.Context, output io.Writer, repo restic.Repository, node *restic.Node) error {
 	var (
-		buf []byte
-		err error
+		buf       []byte
+		ok        bool
+		err       error
+		blobCache *blobcache.BlobCache
 	)
+
+	blobCache = blobcache.NewBlobCache(64 << 20)
```

Comment:
I'd prefer to use `blobCache := ...` here. Instead of explicitly declaring the variable before.

Reply:
ack, updated in current version